test/services.oauth.js

```diff
@@ -0,0 +1,126 @@
+var OAuth = require('../services/oauth');
+var request = require('superagent');
+var assert = require('assert');
+var time = require('timemachine');
```

Comment:
I straight up could not get lolex to work. Also, the only proper choice of mock timestamp is obviously the event of the world's first temporal displacement, so I fixed that too.